src/app/modules/item/pages/item-current-situation/item-current-situation.component.ts

```diff
@@ -92,8 +74,17 @@ export class ItemCurrentSituationComponent implements OnChanges {
     }));
   }
 
-  getHideSelection(): boolean {
-    return !this.watchedGroup && !!this.type && [ 'Task', 'Course' ].includes(this.type);
+  getHideSelection(type: ItemType): boolean {
+    return !this.watchedGroup && !!type && [ 'Task', 'Course' ].includes(type);
```

Comment:
why `&& !!type` ? It cannot be undefined, null or empty.

Reply:
Fixed